### DISCUSSION TITLE
User's completed kata unsolved again

### DISCUSSION THREAD

**Discussion by Pdzoc** [codewars/codewars.com#2698] [General]
Hi!

Some kata's which I solved long ago are still visible in list of tasks that I have NOT completed. I did use Codewars api to check list of katas - one example was solved few years ago, but today is marked as "pure" - without any attempt.

**Comment by Blind4Basics** [comment]
Hi,

* Could you give links to the related kata, please?
* Did you check if your old solution is still valid?
* are you talking about results in the kata search page or your list of "unfinished kata", in your user profile? (like: https://www.codewars.com/users/.../unfinished_solutions )

**Reply by Pdzoc** [reply]
I saw that kata among results in search page with filter "don't completed" kata. 

It looks like there is no my own solution (Solutions -> My Solutions: "There are no solutions yet for this kata."), although list from api includes information that I completed this kata in 2019: `{"id":"51f9d93b4095e0a7200001b8","name":"How many lightsabers do you own?","slug":"how-many-lightsabers-do-you-own","completedLanguages":["javascript"],"completedAt":"2019-08-10T08:34:47.687Z"}` and I see button "Train again".

Link to kata: https://www.codewars.com/kata/51f9d93b4095e0a7200001b8/javascript (id from link and from my list are the same)
And my list including info about this kata: https://www.codewars.com/api/v1/users/Bunkierski/code-challenges/completed?page=6

It was really simple 8kyu kata, I don't suppose that my solution was outdated and therefore removed by admin/moderator.

**Reply by Blind4Basics** [reply]
what about now?

**Reply by Pdzoc** [reply]
I still see few katas as unresolved (one kata after solving and gaining points few days ago was even not added on my list of completed katas), but after all it's not so big amount which could bring a mess. I didn't loose any points so the system is still fair, although it's interesting whether other users noticed similar problem and why the "problem" occurs more often than in the past.

**Reply by Blind4Basics** [reply]
yeah, there are a lot of other entries that look problematic in the records of your account. I only handled the one you talked about, to check nothing "abnormal" was involved. You're facing an old bug, actually. It happend a lot more back in the days, so you surely will encounter it again. But as you said, it's only a "visibility issue". All other stats are compliant.
